cfgov/unprocessed/js/modules/multi-select.js

```diff
@@ -0,0 +1,494 @@
+'use strict';
+
+function init() {
+  var _ = function ( elem, opts ) {
+    var self = this;
+
+    this.elem = $( elem );
+    opts = opts || {};
+
+    configure.call( this, {
+      minChars:  3,
+      maxItems:  10,
+      autoFirst: false,
+      filter: _.FILTER_CONTAINS
+    }, opts );
+
+    this.index = -1;
+    this.name = this.elem.name;
+    this.skipBlur = false;
+    this.optionData = [];
+    this.selections = [];
+    this.options = this.elem.options || [];
+    this.placeholder;
```

Comment:
Move these to private properties of the constructor and drop the `this` reference lookup. E.g.:

```js
function MultiSelect() {
    var _index = -1;
    var _name;

    …

    function init( elem ) {
         _name = elem.name;

         …

         return this;
    }

    …
    
    this.init = init;
    return this;
}
```

The use of `this` in that snippet is only for setting the methods on the returned instance and for returning the instance from those methods to support method chaining (`multiSelect.init( elem ).doSomething().doSomethingElse();`).